### PR TITLE
verify setup for browser

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do?
Adds a section explaining how to verify APM <> RUM have been correctly configured.

### Motivation
[DOCS-5584](https://datadoghq.atlassian.net/browse/DOCS-5584)

### Additional Notes
Verified with @keithbradshaw . Ready to merge.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-5584]: https://datadoghq.atlassian.net/browse/DOCS-5584?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by cswatt — APPROVED** [review]

**Comment by rtrieu** [comment]
@DataDog/rum-browser Can you please review this PR? I was especially not sure if the steps are exactly the same for mobile app SDKs - my guess is the steps / wording would be slightly different.

File: content/en/real_user_monitoring/connect_rum_and_traces/_index.md [hunk 1]
```diff
@@ -273,6 +273,33 @@ To start sending just your iOS application's traces to Datadog, see [iOS Trace C
...
+
+{{% /tab %}}
+
+{{% tab "Mobile" %}}
+
+1. Visit a page in your application.
+2. In your browser's developer tools, go to the **Network** tab.
+3. Check the request headers for a RUM resource and verify that the [required headers are set by the SDK][1].
+
+[1]: /real_user_monitoring/connect_rum_and_traces?tab=browserrum#how-are-rum-resources-linked-to-traces
```
Comment (xgouchet) [inline content/en/real_user_monitoring/connect_rum_and_traces/_index.md:298]: This whole section doesn't make sense for Mobile Apps. I'm guessing the Verifying setup instructions are going to be different for each platform. E.g. for Android : 

---

1. Run your application form Android Studio
2. Visit a screen in your application.
3. Open Android Studio's [**Network Inspector**][1].
4. Check the request headers for a RUM resource and verify that the [required headers are set by the SDK][2].

[1]: https://developer.android.com/studio/debug/network-profiler#network-inspector-overview
[2]: https://docs.datadoghq.com/real_user_monitoring/connect_rum_and_traces?tab=androidrum#how-are-rum-resources-linked-to-traces

---

**Review by xgouchet — CHANGES_REQUESTED** [review]

File: content/en/real_user_monitoring/connect_rum_and_traces/_index.md (hunk 1 above)
Comment (rtrieu) [reply content/en/real_user_monitoring/connect_rum_and_traces/_index.md]: Hi @xgouchet, I had a feeling that would be case, so I'm glad I asked! Would you / @DataDog/rum-mobile be able to help fill in the blanks for the remaining SDKs, as I don't have much insight into those? I did my best guess for iOS.

Comment (xgouchet) [reply content/en/real_user_monitoring/connect_rum_and_traces/_index.md]: I'll be PTO from tomorrow until August 7th, but I'll pass the word around ;)

Comment (rtrieu) [reply content/en/real_user_monitoring/connect_rum_and_traces/_index.md]: @xgouchet That works - thanks very much! :D

File: content/en/real_user_monitoring/connect_rum_and_traces/_index.md [hunk 2]
```diff
@@ -270,6 +270,79 @@ To start sending just your iOS application's traces to Datadog, see [iOS Trace C
...
+{{% /tab %}}
+{{% tab "Roku" %}}
+
+1. Run your application from [software].
+2. Visit a screen in your application.
+3. Open [software]'s [Network Connections and HTTP Traffic instrument][1].
+4. Check the request headers for a RUM resource and verify that the [required headers are set by the SDK][2].
+
+[1]: 
+[2]: https://docs.datadoghq.com/real_user_monitoring/connect_rum_and_traces/?tab=datadog#how-are-rum-resources-linked-to-traces
```
Comment (xgouchet) [inline content/en/real_user_monitoring/connect_rum_and_traces/_index.md:343]: For Roku, there's no way to easily check the outgoing requests (I've tried recently and lost 5 hours for nothing), so we can probably drop this tab

File: content/en/real_user_monitoring/connect_rum_and_traces/_index.md [hunk 3]
```diff
@@ -270,6 +270,79 @@ To start sending just your iOS application's traces to Datadog, see [iOS Trace C
...
+3. Open Android Studio's [Network Inspector][1].
+4. Check the request headers for a RUM resource and verify that the [required headers are set by the SDK][2].
+
+[1]: https://developer.android.com/studio/debug/network-profiler#network-inspector-overview
+[2]: https://docs.datadoghq.com/real_user_monitoring/connect_rum_and_traces?tab=androidrum#how-are-rum-resources-linked-to-traces
+
+{{% /tab %}}
+{{% tab "iOS" %}}
+
+1. Run your application from XCode.
```
Comment (ncreated) [inline content/en/real_user_monitoring/connect_rum_and_traces/_index.md:304]:
```suggestion
1. Run your application from Xcode.
```

File: content/en/real_user_monitoring/connect_rum_and_traces/_index.md [hunk 4]
```diff
@@ -270,6 +270,79 @@ To start sending just your iOS application's traces to Datadog, see [iOS Trace C
...
+2. Visit a screen in your application.
+3. Open XCode's [Network Connections and HTTP Traffic instrument][1].
```
Comment (ncreated) [inline content/en/real_user_monitoring/connect_rum_and_traces/_index.md:306]:
```suggestion
3. Open Xcode's [Network Connections and HTTP Traffic instrument][1].
```

File: content/en/real_user_monitoring/connect_rum_and_traces/_index.md [hunk 5]
```diff
@@ -273,6 +273,67 @@ To start sending just your iOS application's traces to Datadog, see [iOS Trace C
...
+{{% /tab %}}
+{{% tab "React Native" %}}
+
+1. Run your application from [software].
+2. Visit a screen in your application.
+3. Open [software]'s [Network Connections and HTTP Traffic instrument][1].
+4. Check the request headers for a RUM resource and verify that the [required headers are set by the SDK][2].
+
+[1]: 
+[2]: https://docs.datadoghq.com/real_user_monitoring/connect_rum_and_traces/?tab=reactnativerum#how-are-rum-resources-linked-to-traces
```
Comment (louiszawadzki) [inline content/en/real_user_monitoring/connect_rum_and_traces/_index.md:321]: React Native devtools are undergoing a massive revamp, and we're currently in a state where the old devtools is unusable for a lot of projects, and the new one cannot be adopted yet by the majority of projects.

To ensure as many people can follow this and make it easy, I would recommend to use the native tools.

```suggestion
1. Run your application from XCode (respectively Android Studio).
2. Visit a screen in your application.
3. Open Xcode's [Network Connections and HTTP Traffic instrument][1] (resp. Android Studio's [Network Inspector][2]).
4. Check the request headers for a RUM resource and verify that the [required headers are set by the SDK][3].

[1]: https://developer.apple.com/documentation/foundation/url_loading_system/analyzing_http_traffic_with_instruments
[2]: https://developer.android.com/studio/debug/network-profiler#network-inspector-overview
[3]: https://docs.datadoghq.com/real_user_monitoring/connect_rum_and_traces/?tab=reactnativerum#how-are-rum-resources-linked-to-traces
```

**Comment by rtrieu** [comment]
@fuzzybinary @louiszawadzki Thank you so much for your help!

**Review by louiszawadzki — APPROVED** [review]
All good for the React Native part 👍

**Review by fuzzybinary — APPROVED** [review]

**Pull request merged**